packages/components/rate/src/rate.vue

```diff
@@ -195,26 +187,20 @@ function showDecimalIcon(item: number) {
   return showWhenDisabled || showWhenAllowHalf
 }
 
-function emitValue(value: number) {
-  // if allow clear, and selected value is same as modelValue, reset value to 0
-  if (props.clearable && value === props.modelValue) {
-    value = 0
-  }
-
-  emit(UPDATE_MODEL_EVENT, value)
-  if (props.modelValue !== value) {
-    emit('change', value)
-  }
-}
-
```

Comment:
What is the reason to roll it back?

Reply:
code conflict, merge misoperated, I'll add it